com.synology.SynologyDrive.metainfo.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.synology.SynologyDrive</id>
+  <project_license>LicenseRef-proprietary=https://www.synology.com/en-global/company/legal/terms_EULA</project_license>
+  <name>Synology Drive</name>
+  <developer_name>Synology</developer_name>
+  <url type="homepage">https://www.synology.com</url>
+  <summary>Powerful private cloud storage with no recurring fees</summary>
+  <description>
+    <p>Now, Synology makes it possible for every home and office to host your own cloud – with 100% data ownership and no subscription fees. See how this changes the way we live and work, access and share data.</p>
```

Comment:
are you from Synology? If not can you add a disclaimer like `NOTE: This wrapper is not verified by, affiliated with, or supported by Synology Inc.` as a new paragraph here.

Reply:
ok